### PR TITLE
Cleanup First Commit

### PR DESCRIPTION
**Pull request by DGladwell**

### CODE REVIEW

File: cypress/fixtures/Test_Suite_2_ExportVariables.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//Xbox One Game Purchase Test Suite 2
+const searchBar = '#page-header-search-input';
+const xboxTab = '#navbar-tab-8001063';
+
+//PlaysStation Purchase Test Suite 2
+const ps4Tab = '#navbar-tab-8001062';
+const spiderManPS4 = '#main_frame > div.standard-padding-bottom > div > ul.products-row.products-row-cell-4col-4.products-row-1 > li.cell.cell-4col.cell-1 > div > div > div > div.image > table > tbody > tr > td > a > img';
```
Comment (ahmf200) [inline cypress/fixtures/Test_Suite_2_ExportVariables.js]: Is there a way to shorten this selector or make it better? Doesn't look that nice.

File: cypress/fixtures/Test_Suite_3_ExportVariables.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+const amazonSearchButton = '.nav-search-submit';
+const addToBasketButton = '#add-to-cart-button';
+const editBasket = '#hlb-view-cart-announce';
+
+module.exports = {
+
+amazonSearchBar,
+amazonSearchButton,
+addToBasketButton,
+editBasket,
```
Comment (ahmf200) [inline cypress/fixtures/Test_Suite_3_ExportVariables.js]: Indentation.

File: cypress/fixtures/CSS_Test_XPath_Test_ExportVariables.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+//CSS Test Comparison
```
Comment (ahmf200) [inline cypress/fixtures/CSS_Test_XPath_Test_ExportVariables.js]: Please name your files appropriately :) Something like Test_suite_3_variables isn't telling what it contains. Try to name it for example like this -> purchasePageDefinitions. Something along those lines.

File: cypress/fixtures/Test_Suite_4_ExportVariables.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+    fireTabletPanel,
+    fireTabletTen,
+    fireTabletEight,
+ 
+    booksPanel,
+    booksandAudibletab,
+    kindleUnlimitedPage,
+  
+    sportsAndOutdoors,
```
Comment (ahmf200) [inline cypress/fixtures/Test_Suite_4_ExportVariables.js]: Whitespace isn't needed here.

File: cypress/fixtures/Test_Suite_5_ExportVariables.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
+//New York Hotel Search Variables
```
Comment (ahmf200) [inline cypress/fixtures/Test_Suite_5_ExportVariables.js]: You could probably split this out further so you have for example, your search page or fragment.

File: cypress/integration/Tests/Test_Suite_1_AutomationPractice.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
+const atp = require('../../fixtures/Test_Suite_1_ExportVariables');
```
Comment (ahmf200) [inline cypress/integration/Tests/Test_Suite_1_AutomationPractice.js]: What does atp mean?

File: cypress/integration/Tests/Test_Suite_2_Base.com_Game_Purchases.js [hunk 7]
```diff
@@ -1,26 +1,25 @@
-const php = require('../../fixtures/exportVariables');
-
+const bgs = require('../../fixtures/Test_Suite_2_ExportVariables');
```
Comment (ahmf200) [inline cypress/integration/Tests/Test_Suite_2_Base.com_Game_Purchases.js]: What does bgs mean? ;D

File: cypress/integration/Tests/Test_Suite_3_Amazon_Heroes.js [hunk 8]
```diff
@@ -1,26 +1,27 @@
+const amz = require('../../fixtures/Test_Suite_3_ExportVariables');
```
Comment (ahmf200) [inline cypress/integration/Tests/Test_Suite_3_Amazon_Heroes.js]: What does amz mean? ;D

File: cypress/integration/Tests/amazonDropDownOptions.js [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+describe("Hover Functionality", function() {
+    beforeEach(function(){
+        cy.visit("https://www.amazon.co.uk")
+        cy.get(amazonDropDown.shopByDepartmentDropDown).trigger('mouseover') 
+   })
+
+  it("Amazon Pantry", function(){
+    cy.get(amazonDropDown.pantryPanel).trigger('mouseover')
+    cy.xpath(amazonDropDown.amazonPantryStore).click()
+   })
```
Comment (kieranHester) [inline cypress/integration/Tests/amazonDropDownOptions.js]: assert something has happened

File: cypress/integration/Tests/amazonDropDownOptions.js [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+  it("Fire Tablets", function(){
+     cy.get(amazonDropDown.fireTabletPanel).trigger('mouseover')
+     cy.xpath(amazonDropDown.fireTabletTen).click()
+     cy.go('back')
+     cy.get(amazonDropDown.shopByDepartmentDropDown).trigger('mouseover')
+     cy.get(amazonDropDown.fireTabletPanel).trigger('mouseover')
+     cy.xpath(amazonDropDown.fireTabletEight).click()
```
Comment (kieranHester) [inline cypress/integration/Tests/amazonDropDownOptions.js]: same here

File: cypress/integration/Tests/amazonDropDownOptions.js [hunk 11]
```diff
@@ -0,0 +1,34 @@
+const amazonDropDown = require('../../fixtures/amazonDropDownOptionsVariables');
+describe("Hover Functionality", function() {
+    beforeEach(function(){
+        cy.visit("https://www.amazon.co.uk")
+        cy.get(amazonDropDown.shopByDepartmentDropDown).trigger('mouseover') 
+   })
+
+  it("Amazon Pantry", function(){
```
Comment (kieranHester) [inline cypress/integration/Tests/amazonDropDownOptions.js]: names should be describing the test

File: cypress/integration/Tests/amazonDropDownOptions.js [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+     cy.go('back')
+     cy.get(amazonDropDown.shopByDepartmentDropDown).trigger('mouseover')
+     cy.get(amazonDropDown.fireTabletPanel).trigger('mouseover')
+     cy.xpath(amazonDropDown.fireTabletEight).click()
+   })
+
+  it("Books and Audible", function(){
+     cy.get(amazonDropDown.booksPanel).trigger('mouseover')
+     cy.xpath(amazonDropDown.booksandAudibletab).click()
+     cy.reload()
```
Comment (kieranHester) [inline cypress/integration/Tests/amazonDropDownOptions.js]: is this needed

File: cypress/integration/Tests/phpHotelAndFlightBooking.js [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+      cy.get(phpHotel.dateOut).click()
+      cy.xpath(phpHotel.nextMonthArrow).click()
+      cy.xpath(phpHotel.thirtyFirstOfMay).click()
+      cy.get(phpHotel.twentyNinthOfJune).click()
+      cy.get(phpHotel.searchButton).click()
+      cy.get(phpHotel.rendezvousHotel).should('be.visible')
+    })
+
+  it("Flight Booking", function(){
+     cy.get(phpFlight.selectFlighttab).click()
```
Comment (kieranHester) [inline cypress/integration/Tests/phpHotelAndFlightBooking.js]: all we are doing is clicking

File: cypress/integration/Tests/xpathPresentationTest.js [hunk 14]
```diff
@@ -0,0 +1,12 @@
+const xpt = require('../../fixtures/cssXPathPresentationVariables');
+
+describe("CSS vs Xpath", function() {
+    beforeEach(function(){
+        cy.visit("https://www.amazon.co.uk") 
+    })
+    
+    it("XPath Test", function(){
```
Comment (kieranHester) [inline cypress/integration/Tests/xpathPresentationTest.js]: dont know what these are for

File: cypress/integration/Tests/phpHotelAndFlightBooking.js (hunk 13 above)
Comment (DGladwell) [reply cypress/integration/Tests/phpHotelAndFlightBooking.js]: Added assertions

File: cypress/fixtures/amazonDropDownOptionsVariables.js [hunk 15]
```diff
@@ -0,0 +1,34 @@
...
+
+//Fire Tablets Test Suite 4
+const fireTabletPanel = '[data-nav-panelkey="KindleFireTabletPanel"]';
+const fireTabletTen = '//*[@id="nav-flyout-shopAll"]/div[3]/div[6]/div[1]/div/a[3]/span[1]';
+const fireTabletEight = '//*[@id="nav-flyout-shopAll"]/div[3]/div[6]/div[1]/div/a[2]/span[1]';
+
+//Books and Audible Test Suite 4
+const booksPanel = '[data-nav-panelkey="BooksPanel"]';
+const booksandAudibletab = '//*[@id="nav-flyout-shopAll"]/div[3]/div[9]/div[1]/a[3]/span[1]';
+const kindleUnlimitedPage = 'https://www.amazon.co.uk/kindle-dbs/hz/signup?_encoding=UTF8&ref_=nav_shopall_ods_books_con_ku';
```
[the remark this answers is not in the document]
Comment (DGladwell) [reply cypress/fixtures/amazonDropDownOptionsVariables.js]: I remember we had some fun using .json files last time - perhaps we could go through it again sometime next week?

**Pull request merged**